### ISSUE TITLE
Cannot filter messages in console of Chrome DevTools

### ISSUE THREAD

**Issue by klemensz** [dbfannin/ngx-logger#269]
Hi,

This seems to affect only Chrome (testing with latest version 95), but not Firefox, for example. If I have these log messages for example:

<img width="855" alt="Bildschirmfoto 2021-11-02 um 17 35 09" src="https://user-images.githubusercontent.com/1116517/139907666-ef574194-8b66-4309-9e71-181c6c89968d.png">

Then searching for "Initializing" won't give any results:

<img width="854" alt="Bildschirmfoto 2021-11-02 um 17 36 41" src="https://user-images.githubusercontent.com/1116517/139907716-5934c297-1d82-4567-9eab-140fbdf69f95.png">

Searching for "analytics.service" on the other hind will give the relevant results:

<img width="854" alt="Bildschirmfoto 2021-11-02 um 17 36 59" src="https://user-images.githubusercontent.com/1116517/139907742-4ac57de9-1c1a-486c-9e4b-eb983b60e48d.png">

It looks like it's only filtering the file names, but not the actual logging output message?

This has worked before and we haven't updated the library version recently (using v4.2.1). Maybe this issue was introduced by a recent version of Chrome?

**Comment by bmtheo** [comment]
It looks like it is an issue with Chrome indeed

You can reproduce with this line :
```
console.info('%c17:33:05.583 INFO [./src/app/services/analytics.service.ts:41]', 'color:gray', 'Initializing AnalyticsService Provider');
```

From what I can tell, Chrome is not filtering arguments of console logs anymore

**Comment by klemensz** [comment]
I tested with more browsers now:

* Safari (15.1)
* Firefox (93.0)
* Opera (80)

In all of these browsers I could not reproduce the issue.

**Comment by bmtheo** [comment]
Thanks for your tests, I didn't find any issue on Chrome tracker

By any chance if you can open one that'd be great

**Comment by klemensz** [comment]
Do you mean the Chromium bug tracker? https://bugs.chromium.org/p/chromium/issues/list

**Comment by bmtheo** [comment]
Yes, that way they can tell us if it's a bug or if it's expected behavior (to me it looks like a bug but we never know)

Depending on their answer we will see if we need to do something or not

**Comment by klemensz** [comment]
I just noticed the same behavior now in Opera 81.0 which seems to have the Chrome 95 DevTools under the hood.

**Comment by sergiocapozzi77** [comment]
I've opened the Issue 1271075...not sure if someone had already opened it.

**Comment by bmtheo** [comment]
amazing thanks, just adding the complete link if someone is not familiar with crbug
https://crbug.com/1271075

tips if you want it to draw more attention you can "star" the issue (top left corner), you also receive notifications that way (@klemensz)

**Comment by klemensz** [comment]
Thanks, I just starred it and also added a link back to this issue here.

**Comment by bmtheo** [comment]
Chromium stated that this was a bug

They are fixing it (see more info on this issue : https://bugs.chromium.org/p/chromium/issues/detail?id=1263070)

I'm closing this one
